match.py

```diff
@@ -0,0 +1,5 @@
+import requests 
+
+r = requests.get("https://github.com", verify=False)
```

Comment:
[Disabled certificate validation](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS114352.md#DS130821)
Extend default certificate validation, but do not disable or override default rules.

*Fix Guidance:* Always use a valid certificate, even during testing.


Suggestions:
[1] r = requests.get("https://github.com", verify=True)